bin/cron/summer_workshop_enrollments_to_gdrive

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+require_relative '../../lib/cdo/only_one'
+exit unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/google/sheet'
+
+#
+# Uses a Google Cloud service account to write enrollment data for this year into
+# a spreadsheet in Google Drive (with permissions locked down to our organization)
+# in order to send packages to teachers attending 5-day workshops.
+#
+
+EARLIEST_WORKSHOP_END_DATE = Date.new(2020, 6, 1)
+LATEST_WORKSHOP_START_DATE = Date.new(2020, 8, 31)
+
+def update_tabs(sheet)
+  # Seed results array with a header row
+  results = [%w(teacher_email teacher_name workshop_id
+                workshop_start_date workshop_course regional_partner)]
+
+  workshop_ids_this_year = Pd::Workshop.
+    where(subject: Pd::SharedWorkshopConstants::SUBJECT_SUMMER_WORKSHOP).
+    scheduled_end_on_or_after(EARLIEST_WORKSHOP_END_DATE).
+    scheduled_start_on_or_before(LATEST_WORKSHOP_START_DATE).
+    pluck(:id)
+
+  enrollments_this_year = Pd::Enrollment.
+    includes(:workshop).
+    where(pd_workshop_id: workshop_ids_this_year)
+
+  enrollments_this_year.find_each do |enrollment|
+    results << [
+      enrollment.email,
+      enrollment.first_name + ' ' + enrollment.last_name,
+      enrollment.workshop.id,
+      enrollment.workshop.workshop_starting_date.to_date,
+      enrollment.workshop.course,
+      enrollment.workshop.regional_partner&.name
+    ]
+  end
+
+  sheet.export(
+    sheet_name: 'enrollments',
+    rows: results
+  )
+end
+
+def notify_of_external_sharing(sheet)
+  if sheet.external_emails_with_access.present?
+    email_domains = external_emails.map {|email| email.slice(/@.*/)}.uniq
```

Comment:
it might be nice as a follow-up to also move this "email domains" logic into somewhere reusable

Reply:
Totally, I'll follow up to get the other script to use the `Google::Sheet` class and look at including this.